docs/private-dns/api-reference.md

```diff
@@ -0,0 +1,561 @@
+---
+title: API Reference
+sidebar_position: 3
+toc_min_heading_level: 3
+toc_max_heading_level: 4
+---
+
+<!-- 
+    The content below is simply an automatic conversion from the OpenAPI spec
+    https://api.adguard-dns.io/static/swagger/openapi.json to markdown using
+    https://swagger-markdown-ui.netlify.app/.
+
+    If you want to change it, ask the developers to change the OpenAPI spec.
+-->
+
+# AdGuard DNS API
+DNS API documentation
+
+## Version: 1.3
```

Comment:
Update version to 1.4. API was updated recently (added custom blocking mode option) and I forgot to bump version in repo. Also maybe we need to add CHANGELOG here?

Reply:
It's autogenerated from https://api.adguard-dns.io/static/swagger/openapi.json

Reply:
Changelog is a good idea. Can you make it available on `api.adguard-dns.io`?

Reply:
Done. New version at https://api.adguard-dns.io/static/swagger/openapi.json and [changelog](https://api.adguard-dns.io/static/api/CHANGELOG.md)